todo_app/app.py

```diff
@@ -1,26 +1,32 @@
 from flask import Flask, redirect, request
 from flask import render_template
-from todo_app.data.trello_items import get_list_organised_items, get_lists, add_item, update_item_status
+from todo_app.data.trello_items import get_items, get_lists, add_item, update_item_status
 
+from todo_app.domain.view_model import ViewModel
 from todo_app.flask_config import Config
 
-app = Flask(__name__)
-app.config.from_object(Config())
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
 
+    @app.route('/')
+    def index():
+        items = get_items()
+        lists = get_lists()
+        view_model = ViewModel(items, lists)
```

Comment:
Minor (and tbh probably not worth changing now!), but worth noting that you can get most of this work done for you if you use the `/lists`  endpoint with `?cards=open`, which returns the lists with their associated cards which saves us one API call and the effort of combining the two